server.js

```diff
@@ -68,6 +68,22 @@ function getClasses(raceClasses) {
   return Promise.all(classesPromises);
 }
 
+app.get('/api/v1/faction/:name', async (request, response) => {
+  try {
+    const races = await database('races').where('faction', request.params.name).select();
+    if (races.length) {
+      response.status(200).json(races);
+    } else {
+      response.status(404).json({
+        error: `Could not find any race from ${request.params.name}`
+      });
```

Comment:
3. Return error if there is no races connected with faction in DB as response with status code `404`